msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -111,8 +114,15 @@ public AcquireTokenResult acquireToken(@NonNull final AcquireTokenOperationParam
             throw authorityResult.getClientException();
         }
 
+        // Build up params for Strategy construction
+        final OAuth2StrategyParameters strategyOptions = new OAuth2StrategyParameters();
+        strategyOptions.setContext(parameters.getAppContext());
+
         //1) Get oAuth2Strategy for Authority Type
-        final OAuth2Strategy oAuth2Strategy = parameters.getAuthority().createOAuth2Strategy();
+        final OAuth2Strategy oAuth2Strategy = parameters
+                .getAuthority()
+                .createOAuth2Strategy(strategyOptions);
```

Comment:
I'm more inclined to just pass parameter.getAppContext() here. (I assume that you introduce this for future-proofing) 

Your call.

Reply:
I went back-and-forth on this quite a bit....and like you I'm not totally satisfied with it. I ultimately went with this approach for two reasons:
1. Future proofiness (as you call out)
2. Not all the places we construct a `Strategy` do we have a `Context` available -- we could annotate this as a `@Nullable` parameter but I thought this might actually be weirder and less flexible

Does that strike you as a reasonable enough reason to leave-in this idea of `StrategyOptions`?

Reply:
I'm fine with that.